assets/addon_browser/addons.conf

```
@@ -520,4 +520,17 @@
 	  "github": "https://github.com/Arctic-Gaming-LLC/Heatmap"
     }
   }
+  {
+    name: "Konquest"
+    description: "Town claiming and capturing with global kingdoms, resident management and stats/titles. Displays territory regions using BlueMap."
+    author: "Rumsfield"
+    apiVersion: "2"
+    platforms: ["spigot", "paper", "purpur"]
```

```suggestion
    platforms: ["spigot", "paper"]
```
`purpur` is not a vaild platform for this config rn, i think it's clear that it supports purpur if spigot and paper are mentioned :)